1.cellprofiler_ic/image_quality_control/scripts/2.qc_report.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
```

Comment:
I'm confused, this looks like an R file, but the file has the `.py` extension and the python shebang

Reply:
LOL oops, that's an error on my end. Thanks for catching that!

Reply:
Fixed!